src/main/java/com/team4/testingsystem/controllers/QuestionController.java

```diff
@@ -0,0 +1,25 @@
+package com.team4.testingsystem.controllers;
+
+import com.team4.testingsystem.entities.Question;
+import com.team4.testingsystem.services.QuestionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "/question")
+public class QuestionController {
+    private final QuestionService questionService;
+
+    @Autowired
+    public QuestionController(QuestionService questionService) {
+        this.questionService = questionService;
+    }
+
+    @GetMapping("/{id}")
+    public Question getLevel(@PathVariable("id") Long id) {
+        return questionService.getQuestionById(id);
+    }
```

Comment:
I think we need an ability to create, update or delete question as well

Reply:
do we need to delete questions? they should be used in tests and then to display hr. If we want to delete, then we need to check if this question is used in tests?

Reply:
Yep, you are right. Alternatively, we can block the question. Anyway, it will be a separate task.
Still, we need a way to create and update the questions